<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuestScreen</class>
 <widget class="QWidget" name="GuestScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1366</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1366</width>
    <height>768</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1366</width>
    <height>768</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Century Gothic</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>GuestScreen</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#backGroundImage{
background-image: url(:/images/images/frmBackGround.png);
}

#GuestScreen {
  border: 0 px
}
</string>
  </property>
  <widget class="QLabel" name="CustomerLogo">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>385</width>
     <height>81</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="pixmap">
    <pixmap>images/default_logo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="deviceStatusLabel">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>2</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/svg/resources/led_on.svg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="deviceStatusStrLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>3</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>HID Device connected</string>
   </property>
  </widget>
  <widget class="QLabel" name="instructionLabel">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>25</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgba(255, 255, 255, 0);
color: rgb(0, 174, 239);</string>
   </property>
   <property name="text">
    <string>Present ID to Access</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="kabDescription">
   <property name="geometry">
    <rect>
     <x>56</x>
     <y>231</y>
     <width>374</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 174, 239);
background: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>#DisplayDescriptionHere#</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="kabSerial">
   <property name="geometry">
    <rect>
     <x>56</x>
     <y>262</y>
     <width>374</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600; color:#00aeef;&quot;&gt;Serial Number&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 174, 239);
background: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>#DisplaySerialNumberHere#</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pnlBand">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>270</y>
     <width>633</width>
     <height>273</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>633</width>
     <height>273</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>40</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgba(0, 246, 0);
color: rgba(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>ALL TOOLS IN</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="toolsStatusButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>270</y>
     <width>631</width>
     <height>271</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>40</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TOOLS OUT</string>
   </property>
  </widget>
  <widget class="QWidget" name="tailWidget" native="true">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>70</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="backgroundImage" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>768</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#backgroundImage{
background-image: url(:/images/images/frmBackGround.png);
}
</string>
   </property>
   <widget class="QPushButton" name="switchButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>450</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/images/images/null.png</normaloff>
      <normalon>:/images/images/null.png</normalon>:/images/images/null.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>128</width>
      <height>128</height>
     </size>
    </property>
    <property name="shortcut">
     <string>Ctrl+B</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblDate">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>30</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(172, 172, 172);
background: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Date:  16/12/2016</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTime">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>55</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(172, 172, 172);
background: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Time:  5:57:10:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="about">
    <property name="geometry">
     <rect>
      <x>1310</x>
      <y>5</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>8</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgba(255, 255, 255, 0);
font: 75 8pt &quot;Century Gothic&quot;;
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>LOCKED</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="vesionNumber">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>720</y>
      <width>423</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Century Gothic&quot;;color: rgb(255, 255, 255);
background: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>#DisplayVersionNumberHere#</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="topLeftLabel">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>1</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/images/logo.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="FirmwareVersion">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>740</y>
      <width>423</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Century Gothic&quot;;
color: rgb(255, 255, 255);
background: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>#DisplayFirmwareVersionHere#</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="powerSts">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>0</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgba(255, 255, 255, 0);
font: 75 8pt &quot;Century Gothic&quot;;
color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <zorder>backgroundImage</zorder>
  <zorder>CustomerLogo</zorder>
  <zorder>deviceStatusLabel</zorder>
  <zorder>deviceStatusStrLabel</zorder>
  <zorder>instructionLabel</zorder>
  <zorder>kabDescription</zorder>
  <zorder>kabSerial</zorder>
  <zorder>pnlBand</zorder>
  <zorder>toolsStatusButton</zorder>
  <zorder>tailWidget</zorder>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
